apps/frontend/src/libs/components/icon-button/libs/constants/icon-size.constant.ts

```diff
@@ -0,0 +1,6 @@
+const ICON_SIZE = {
```

Comment:
it seems like an enum

Reply:
Hm. So you suggest to move it into `apps/frontend/src/libs/enums/icon-size.enum.ts`? And I see it like:
```ts
const IconSize = {
  DEFAULT: 20;
} as const;
```

Reply:
Do this the way I described, but if you mean something different, please explain your suggestion